### PR TITLE
[Build] Fix build failure due to recent changes in ROS and NNS @open sesame 10/8 12:33

### PR DESCRIPTION
**Pull request by wooksong**
This patch fixes build failure due to recent changes in ROS and NNS.
In detail,
 - As 'type' is removed from GstTensorMemory [1], this change is also applied to the ROS sink.
 - CMakeLists.txt is revised to support both 'colcon' and 'cmake'.
 - Instead of 'kinetic' that is an EOL ROS distro, 'melodic' is set to a default ROS distro.
 - Bumped up the version of NNStreamer to 1.6.0

[1] https://github.com/nnstreamer/nnstreamer/commit/63d6fba3e441c7c514713c28f6309d8c013ccbee

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

**Comment by wooksong** [comment]
Although it passed the Ubuntu build checker in the CI system, ```pdebuild``` on the later distros such as ```bionic``` and ```focal``` will fail. This issue will be resolved in #41.

File: debian/changelog [hunk 1]
```diff
@@ -1,5 +1,5 @@
-nnstreamer-ros (2019.2.29) unstable; urgency=medium
+nnstreamer-ros (0.0.1) unstable; urgency=medium
```
Comment (leemgs) [inline debian/changelog:1]: Look good.

File: debian/install [hunk 2]
```diff
@@ -10,5 +10,5 @@
 /opt/ros/kinetic/share/nns_ros_bridge/package.xml
 /opt/ros/kinetic/share/common-lisp/ros/nns_ros_bridge/msg/*
 /opt/ros/kinetic/include/nns_ros_bridge/tensors.h
-/usr/lib/*/gstreamer-1.0/*
+/opt/ros/kinetic/lib/gstreamer-1.0/*
```
Comment (leemgs) [inline debian/install:13]: kinetic or melodic? which one?

Comment (again4you) [reply debian/install:13]: IMO, it looks OK since debian packaging only supports kinetic.

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**